packages/renderer/src/lib/container/ContainerDetailsTerminal.svelte

```diff
@@ -148,8 +148,24 @@ onMount(async () => {
   await executeShellIntoContainer();
 });
 
+function removePromptFromTheEnd(terminal: string): string {
```

Comment:
Any way we could possibly have this in a separate file so it can be used for other Terminal sections? (pods, kubernetes, etc.)

Reply:
Good point, but right now it is only client for terminals store and I assume this is the only place where the function can be used right now. When time comes we might pull it out.

Reply:
ACK. That makes sense and you're right. It's the only place using stores vs Kubernetes implementation.